extra-cmds/readtags-cmd.c

```diff
@@ -672,6 +989,13 @@ extern int main (int argc, char **argv)
 					exit (1);
 				}
 			}
+			else if (strcmp (optname, "canonicalize-path") == 0)
+			{
+				canon = 1;
+				if (!canonicalizePathTable)
+					canonicalizePathTable = hashTableNew (509, hashCstrhash, hashCstreq,
```

Comment:
We will not need 509 anymore because the hashtable will be able to grow its buckets automatically.